src/SDKs/Azure.Base/data-plane/Azure.Base/Response.cs

```diff
@@ -68,6 +70,6 @@ public bool TryGetHeader(string name, out string value)
         public override int GetHashCode() => base.GetHashCode();
 
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public override string ToString() => _message.ToString();
+        public override string ToString() => _httpResponse.ToString();
```

Comment:
Unless I'm misreading, it seems that `_httpResponse` could be null. Should we consider using something like `?.ToString() ?? String.Empty ` to ensure that `ToString` doesn't throw?

Reply:
It shouldn't be null, added a check to the constructor.